I am interested in using your game on my website. Could you add a license file like the MIT one in this pull request so I know how you want your software used?

jpbida opened the pull request.

polygonboutique commented:
Sure. I have updated the license to bsd-2 and added additional documention. 
If you think that there are certain functions missing, let me know.

jpbida commented:
Thanks.

On Thu, Feb 9, 2017 at 11:20 AM, polygonboutique <notifications@github.com>
wrote:

> Sure. I have updated the license to bsd-2 and added additional documention.
> If you think that there are certain functions missing, let me know.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/polygonboutique/MathKit/pull/1#issuecomment-278744194>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGrFN_iif5tyQkVLRDK6wLzbR276VR8ks5ra2cIgaJpZM4L3b2N>
> .
>
